PopcornEditor/src/butter.js

```diff
@@ -922,67 +922,127 @@ window.Butter = {
           finishedCallback( project );
         }
 
+        function initialMediaSources( qs ){
```

Comment:
omigosh, that's a great idea!
yah, in my head i was all "but how will this work/autodetect youtube pieces and more?"
maybe it's a short prefix before each piece like
IA:drascii
YT:dk3lsf3
VM:....
and/or have ability to have entire url query string pertain to (just 1) service and have that service "decode and expand it".
we do something _sort of_ similar now for IA pieces, eg:
(you can see in the "Network" tab)
https://archive.org/services/maker.php?callback=caller1461964785875&url=https%3A%2F%2Farchive.org%2Fdetails%2Fredrake
to expand the expanded IA url/asset to the pieces popcorn needs to issue linkback, get thumbnails, and more